index.html

```diff
@@ -26,7 +27,7 @@
       <input type="button" value="Deposit" />
       <input type="button" value="Withdraw" />
     </div>
-
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
     <script src="atm.js"></script>
```

Comment:
you have to script tags linking jq, you should try to figure out which one works and get rid of the other one. I think it may cause an error trying to load a file that doesn't exist